mlflow/xgboost.py

```diff
@@ -269,31 +250,8 @@ def log_model(
     :param await_registration_for: Number of seconds to wait for the model version to finish
                             being created and is in ``READY`` status. By default, the function
                             waits for five minutes. Specify 0 or None to skip waiting.
-    :param pip_requirements: Either an iterable of pip requirement strings
-        (e.g. ``["scikit-learn", "-r requirements.txt"]``) or the string path to a pip requirements
-        file on the local filesystem (e.g. ``"requirements.txt"``). If provided, this describes the
-        environment this model should be run in. If ``None``, a default list of requirements is
-        inferred from the current software environment. Requirements are automatically parsed and
-        written to a ``requirements.txt`` file that is stored as part of the model. These
-        requirements are also written to the ``pip`` section of the model's conda environment
-        (``conda.yaml``) file.
-    :param extra_pip_requirements: Either an iterable of pip requirement strings
-        (e.g. ``["scikit-learn", "-r requirements.txt"]``) or the string path to a pip requirements
-        file on the local filesystem (e.g. ``"requirements.txt"``). If provided, this specifies
-        additional pip requirements that are appended to a default set of pip requirements generated
-        automatically based on the user's current software environment. Requirements are also
-        written to the ``pip`` section of the model's conda environment (``conda.yaml``) file.
-
-        .. warning::
-            The following arguments can't be specified at the same time:
-
-            - ``conda_env``
-            - ``pip_requirements``
-            - ``extra_pip_requirements``
-
-        :ref:`This example<pip-requirements-example>` demonstrates how to specify pip requirements
-        using ``pip_requirements`` and ``extra_pip_requirements``.
-    :param kwargs: kwargs to pass to `xgboost.Booster.save_model`_ method.
+    :param pip_requirements: {{ pip_requirements }}
+    :param extra_pip_requirements: {{ extra_pip_requirements }}
```

Comment:
I only deduplicated `pip_requirements` and `extra_pip_requirements`, but will file follow-up PRs for other parameter in the upcoming sprint.

Reply:
One con is this makes a bit harder to quickly check how arguments work because you need to jump to a different file.

Reply:
It takes 1 ~ 2 ms to complete the replacement.